apps/rush-lib/src/cli/actions/UpdateCloudCredentialsAction.ts

```diff
@@ -63,12 +56,19 @@ export class UpdateCloudCredentialsAction extends BaseRushAction {
         this.rushConfiguration
       );
       terminal.writeErrorLine(
-        `The a build cache has not been configured. Configure it by creating a ` +
+        `The build cache has not been configured. Configure it by creating a ` +
```

Comment:
Is it possible to hit this without having a build-cache.json file?

Reply:
`loadFromDefaultPathAsync()` claims that:
```
* If the file has not been created yet, then undefined is returned.
```